test/python/compiler/test_assembler.py

```diff
@@ -602,6 +602,46 @@ def test_assemble_schedule_enum(self):
         self.assertEqual(test_dict['config']['meas_return'], 'avg')
         self.assertEqual(test_dict['config']['meas_level'], 2)
 
+    def test_assemble_parametric(self):
+        """Test that parametric pulses can be assembled properly into a PulseQobj."""
+        sched = pulse.Schedule(name='test_parametric')
+        sched += pulse.Gaussian(duration=25, sigma=4, amp=0.5j)(DriveChannel(0))
+        sched += pulse.Drag(duration=25, amp=0.2+0.3j, sigma=7.8, beta=4)(DriveChannel(1))
+        sched += pulse.ConstantPulse(duration=25, amp=1)(DriveChannel(2))
+        sched += pulse.GaussianSquare(duration=150, amp=0.2,
+                                      sigma=8, width=140)(MeasureChannel(0)) << sched.duration
+        backend = FakeOpenPulse3Q()
+        backend.configuration().parametric_pulses = ['gaussian', 'drag',
+                                                     'gaussian_square', 'constant']
+        qobj = assemble(sched, backend)
+
+        self.assertEqual(qobj.config.pulse_library, [])
+        qobj_insts = qobj.experiments[0].instructions
+        self.assertTrue(all(inst.name == 'parametric_pulse'
+                            for inst in qobj_insts))
+        self.assertEqual(qobj_insts[0].pulse_shape, 'gaussian')
+        self.assertDictEqual(qobj_insts[0].parameters, {'duration': 25, 'sigma': 4, 'amp': 0.5j})
+        self.assertListEqual(
+            qobj.to_dict()['experiments'][0]['instructions'][0]['parameters']['amp'],
+            [0.0, 0.5])
```

Comment:
To edge on the side of caution, would you mind testing the Drag, ConstantPulse, and GaussianSquare?